_layouts/default.html

```diff
@@ -113,7 +116,10 @@
           <a href="{{ site.baseurl }}/blinky.html">3. blinky</a>
         </li>
         <li>
-          <a href="{{ site.baseurl }}/modules.html">4. modules<!--ul-->
+          <a href="{{ site.baseurl }}/ap.html">4. access point</a>
```

Comment:
I would put this after modules and twitter; modules & node API use seem more core to the Tessel experience.